app/services/checksum_service.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module ChecksumService
```

Comment:
@RudyOnRails You should include this module in `DuplicateUploadVerificationService` and remove the duplicate code. It doesn't matter if the methods are private or not.